components/Event.js

```diff
@@ -188,9 +186,6 @@ class Event extends React.Component {
       };
     }
 
-    const backgroundImage =
-      event.backgroundImage || get(event, 'parentCollective.backgroundImage') || defaultBackgroundImage;
```

Comment:
I may be missing something, but my understanding is that we're loosing the fetch of parent collective `backgroundImage` for events in case `collective.backgroundImage` is null.